source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -1,871 +1,865 @@
+  setting up an update location.
 
-   .. code-block:: bash
+First time setup
+================
 
-      sudo mkdir -p  /etc/nginx/conf.d
+First time setup creates and initializes the workspace used by mixer. Complete
+setup before you create a mix.
 
-#. Copy the default example configuration file. 
+#. Create workspace.
 
-   .. code-block:: bash
+   The mixer tool uses a simple workspace to contain all input and output in a
+   basic directory structure. The workspace is simply an empty folder that you
+   will execute the mixer commands from. Each mix will use its own separate
+   workspace.
 
-      sudo cp -f /usr/share/nginx/conf/nginx.conf.example /etc/nginx/nginx.conf
+#. Initialize the workspace and mix.
 
-#. Open ``mixer.conf`` with an editor.
+   Before you create a mix, you must explicitley initialize the mixer workspace.
+   During initialization, the mixer workspace is configured and the base for
+   your mix is defined. By default, your mix will be based on the latest
+   upstream version and start with the minimum set of bundles. Your first custom
+   mix version number will start at 10. You can alternately select other
+   versions or bundle sets to start from.
 
-   .. code-block:: bash
- 
-      sudo nano /etc/nginx/conf.d/mixer.conf 
+   Initialization creates the directory structure used by mixer and adds the
+   :file:`builder.conf` file, which is used to configure the mixer tool.
 
-#. Add the server configuration content below to ``mixer.conf``. 
+   View the `mixer.init man page`_ for more information on mixer
+   initialization.
 
-   .. code-block:: console
+   View the list of `suitable versions`_ to mix from.
 
-      server {
-           server_name localhost;
-           location / {
-                     root /var/www/mixer;
-                     autoindex on; 
-           }
-      }
+#. Edit builder.conf.
 
-#. Restart the daemon, enable nginx on boot, and start the service.
+   :file:`builder.conf` is used to configure the mixer tool. For example, it
+   allows you to configure where mixer output is located and where swupd update
+   content will be located.
 
-   .. code-block:: bash
+   At minimum, set the URL of your update server so your custom OS knows where
+   to get update content.
 
-      sudo systemctl daemon-reload 
+   Refer to the `builder.conf`_ section for more information.
 
-      sudo systemctl enable nginx 
+Create a mix
+============
 
-      sudo systemctl start nginx 
+A mix is created with the following steps:
 
-#. To verify the web server is running, check it in an Internet browser:
+#. Add custom RPMs and set up local repo (optional).
 
-   .. code-block:: bash
+   If you are adding custom RPMs to your mix, you will need to add the RPMs to
+   your mix workspace and set up a corresponding local repository.
 
-      http://localhost
+   You can use :ref:`autospec<autospec>`, `mock`, `rpmbuild`, or similar tools
```

Comment:
Instead of "You can use :ref:`autospec<autospec>`, `mock`, `rpmbuild`, or similar tools to build RPMs from scratch.", I would suggest "Follow the autospec guide on how to build RPM packages for |CL|."  Suggesting mock, rpmbuild, or similar tools creates confusion.